xbmc/input/linux/LIRC.h

```diff
@@ -18,9 +18,6 @@
  *
  */
 
-#ifndef LIRC_H
-#define LIRC_H
```

Comment:
#pragma once

Reply:
pragma once is still not standard, but widely used.

Reply:
![image](https://user-images.githubusercontent.com/1089309/33928574-a0d41084-dfe6-11e7-871e-f37e93608336.png)

Source: wikipedia. Should be good enough for us, right?

Reply:
It already appears around 1600 times in our codebase.
If we want to avoid it, then the `#ifndef ...`, `#define ...` lines have to be reintroduced.

Reply:
Not really :-) go ahead with pragma once